controllers/channelController.js

```diff
@@ -22,26 +22,34 @@ const { classifyError } = require('../helpers/formatters');
  * Create a new channel
  */
 router.post('/', verifyToken, (req, res) => {
-  Channel.create(req.body).then(channel => {
-    res.json(channel);
-  })
-  .catch(err => {
-    var errRes = classifyError(err);
-    res.status(errRes.status).json(errRes.message);
-  });
+  if(req.user.role && req.user.role.includes("admin")){
+    Channel.create(req.body).then(channel => {
+      res.json(channel);
+    })
+    .catch(err => {
+      var errRes = classifyError(err);
+      res.status(errRes.status).json(errRes.message);
+    });
+  }else{
+    res.status(403);
+  }
 });
 
 /**
  * Get a channel by its id
  */
 router.get('/:id', (req, res) => {
-  Channel.get(req.params.id).then(channel => {
-    res.json(channel);
-  })
-  .catch(err => {
-    var errRes = classifyError(err);
-    res.status(errRes.status).json(errRes.message);
-  });
+  if(req.user.role && req.user.role.includes("admin")){
```

Comment:
I don't think this endpoint should be protected. If we ever want to display channel descriptions or other such info we'll need to fetch data for a specific channel, which every client should be able to do. We can protect the endpoint with a token, fine, but wasn't really ever meant to be something only accessible by admins.